src/shell/dotnet/Shell/Fdc3/ResolverUi/Fdc3ResolverUi.xaml

```diff
@@ -0,0 +1,144 @@
+﻿<!--
+********************************************************************************************************
+Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+Unless required by applicable law or agreed to in writing, software distributed under the License
+is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and limitations under the License.
+********************************************************************************************************
+-->
+<Window x:Class="MorganStanley.ComposeUI.Shell.Fdc3.ResolverUi.Fdc3ResolverUi"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:MorganStanley.ComposeUI.Shell.Fdc3.ResolverUi"
+        mc:Ignorable="d" 
+        WindowStartupLocation="CenterScreen"
+        ResizeMode="CanResizeWithGrip"
+        WindowStyle="None"
+        MouseDown="Window_MouseDown"
+        Background="Transparent"
+        AllowDrop="True"
+        AllowsTransparency="True">
+
+    <Window.Resources>
+        <Style TargetType="Button" x:Key="CancelButtonStyle">
```

Comment:
I'd drop all this styling in favour of a simple but functional look for the moment.
It doesn't match anything we currently do, and it is kind of detrimental to future styling to have these localized custom styles that need to be erased.